### ISSUE TITLE
Spritebuilder crashes with grayscale png files

### ISSUE THREAD

**Issue by matthijn** [spritebuilder/SpriteBuilder#348]
The title gives it away. When adding grayscale (e.g in photoshop -> image -> mode -> grayscale) images the builder crashes when publishing that image. 

Here is a full crash report: https://gist.github.com/Matthijn/079029ab3ebe4f2756bb

**Comment by LearnCocos2D** [comment]
Yup, just got the same crash (context is nil) when publishing with a grayscale image in the project.

**Comment by stevekanter** [comment]
I'm experiencing this same thing, in both the App Store build as well as the 1.1.0-beta.1 build.  Any progress on this issue and/or potential resolution?
